internal/provider/resource_gitlab_group_variable.go

```diff
@@ -25,53 +25,7 @@ var _ = registerResource("gitlab_group_variable", func() *schema.Resource {
 			StateContext: schema.ImportStatePassthroughContext,
 		},
 
```

Comment:
I know it wasn't modified in this PR but I'd suggest changing the description of the resource to match the GitLab API docs like the other suggestions I've made:
```
Description: `The ` + "`" + `gitlab_group_variable` + "`" + ` resource allows to manage the lifecycle of a group-level CI/CD variable.
```